### PR TITLE
Feature/10 search timeline

### LINKED ISSUE #10: Search page - timeline

### PR DESCRIPTION
**Pull request by YoYuUm**
closes #10

### CODE REVIEW

**Comment by YoYuUm** [comment]
7bd6843be8c4140bb7f003f66128815add232083 changes current-year to selected-year so probably I would need to change it here as well...

**Comment by lraivio** [comment]
If you think it's a good change.

**Comment by YoYuUm** [comment]
Yes, I was considering a similar change myself too.

File: app/components/SearchBox.js [hunk 1]
```diff
@@ -13,7 +14,11 @@ class SearchBox extends React.Component {
   }
 
   handleSubmit() {
-    SearchActionCreators.requestSearch(this.state.searchQuery, this.props.filters);
+    let filters = _.clone(this.props.filters);
+    console.log(filters);
```
Comment (lraivio) [inline app/components/SearchBox.js]: Forgotten console.log

File: app/components/SearchTimeline.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
+'use strict';
+
+import React from 'react';
+import ReactSlider from 'react-slider';
+import SearchActionCreators from '../actions/SearchActionCreators';
```
Comment (lraivio) [inline app/components/SearchTimeline.js]: There could be one empty line between node_module imports and our own imports.

File: app/pages/SearchPage.js [hunk 3]
```diff
@@ -50,13 +53,19 @@ class SearchPage extends React.Component {
...
+            years={this.state.years}
+          />
           <SearchControls
             filters={this.state.filters}
             filtersOptions={this.state.filtersOptions}
             view={this.state.view}
           />
-          <div className='search-timeline'></div>
+          <div className='container'>
+              <SearchTimeline years={this.state.years} />
```
Comment (lraivio) [inline app/pages/SearchPage.js]: Too much indentation here.

**Comment by lraivio** [comment]
Very nice! I really like how the timeline looks and functions :+1:

**Comment by YoYuUm** [comment]
I have fixed your comments and add a commit to rename the current-year class names

**Comment by lraivio** [comment]
:+1:

**Pull request merged**